easybuild/easyconfigs/g/GCCcore/GCCcore-10.3.0.eb

```diff
@@ -33,6 +33,7 @@ patches = [
     'GCCcore-6.2.0-fix-find-isl.patch',
     'GCCcore-9.3.0_gmp-c99.patch',
     'GCCcore-9.3.0_nvptx_sm_35_default.patch',
+    'gcc-10.3.0_fix-ice-in-tsubst.patch',
```

Comment:
Do we have a policy on patch names? The typical pattern would the easyconfig name first.

Reply:
For GCCcore there are a mix of patches with names of the part they are patching (`gcc-8.2.0-isl-0.20-missing-include.patch` and the `mpfr-*`) and patches with names starting with `GCCcore`.

Reply:
good enough for me

Reply:
We're not very strict about patch names, main bit is that it should be clear for which version it's intended, and have a descriptive name, so OK to me too in this case.